pages/about/groups/ariawg.md

```diff
@@ -33,11 +31,17 @@ Some information on this page is also shown on, and may be more current in, the
 {% include_cached toc.html type="end" %}
 {:/}
 
-## Announcements
+## Charter
+
+The ARIA Working Group is chartered until January 2027.
+
+To learn about the group’s focus, scope and deliverables, see the [ARIA Working Group Charter](https://www.w3.org/2025/01/aria-charter).
+
+## Current work 
 
-No announcements at the moment.
+For details of the current work, see the [ARIA Working Group wiki](https://www.w3.org/WAI/ARIA/wiki/). (This doesn't seem to be maintained - should we link to a GitHub page?)
 
-## Current Work
+Are we keeping the following 'Roadmap', Deliverables and Status' (out of date?), and 'Work flow' child pages? Should we move any of that information to this section or just continue to link to it? Should we turn the list items below into headings with a brief description (as we did for WCAG current work on the AG WG page)?
```

Comment:
```suggestion
```

I propose we remove these links from here and include some info here (as done in AGWG) and other info on the GH wiki